fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Comparisons.java

```diff
@@ -932,6 +964,53 @@ public Object getComparand(@Nullable FDBRecordStoreBase<?> store, @Nullable Eval
             return context.getBinding(parameter);
         }
 
+        @Nonnull
+        @Override
+        public Comparison rebase(@Nonnull final AliasMap translationMap) {
+            if (internal == Bindings.Internal.CORRELATION) {
+                final var alias = CorrelationIdentifier.of(internal.identifier(parameter));
+                final var translatedAlias = translationMap.getTargetOrDefault(alias, alias);
```

Comment:
This is introducing `var` in ways that I am not certain there is consensus around yet.

Reply:
I am essentially using IntelliJ here. When in Java 11 mode, by default when you introduce a variable it now goes to `var` instead of the static type.  If you don't do that IntelliJ gives you a code-style warning. We can change all that. Is there anything specific about this you don't like?